### ISSUE TITLE
Executing action failed

### ISSUE THREAD

**Issue by sphh** [eyalroz/removedupes#57]
I tried to remove some duplicates with v0.5.2b2. It found the duplicates ok (after the [obligatory restart of TB](https://addons.thunderbird.net/en-US/thunderbird/addon/removedupes/reviews/#review-1166173)), but could not execute the action specified; it could not
- delete the selected email immediately
- could not move them to a different folder on the same IMAP server
- could not move them to a local folder

as mentioned here:
- https://addons.thunderbird.net/en-US/thunderbird/addon/removedupes/reviews/#review-1167486
- https://addons.thunderbird.net/en-US/thunderbird/addon/removedupes/reviews/#review-1167485

After downgrading to v0.5 it worked again.

Thunderbird: 78.11.0 (64-bit) on Linux

**Comment by rsteinmetz70112** [comment]
I am having a similar problem Windows 10 Thunderbird 78.12.0 (64bit) Remove Duplicates 0.5.2b2
Search works but I cannot move messages to trash nor remove them.

I get the following error:

Failed moving messages to: 
mailbox://nobody@Local%20Folders/Trash

**Comment by eyalroz** [comment]
@sphh  : Can you please copy and paste the errors you get in the Error Console, when trying to move the messages to Trash?

Also, you mentioned an obligatory restart; can you please open a separate issue about that?

**Comment by eyalroz** [comment]
@rsteinmetz70112 : What are you seeing in the Error Console?

**Comment by eyalroz** [comment]
Ok, I think I know why this happened. Please try this new build:

[removedupes_0.5.2b3_tbird.xpi.zip](https://github.com/eyalroz/removedupes/files/6896492/removedupes_0.5.2b3_tbird.xpi.zip)

**Comment by sphh** [comment]
I just installed and tried `v0.5.2b3_tbird`. It does work on my installation with an IMAP account.

TB 78.11.0 (64-bit) on Linux Mint 20.2 Cinnamon

NB: The error console shows this error:
```
Prompter: Browser not available or tab modal prompts disabled. Falling back to window prompt. Prompter.jsm:1051
    set modalType resource://gre/modules/Prompter.jsm:1051
    setPropertyAsUint32 resource://gre/modules/Prompter.jsm:1703
    removeDupesFromSingleFolder chrome://removedupes/content/removedupes-common.js:465
    removeDuplicates chrome://removedupes/content/removedupes-common.js:427
    onAccept chrome://removedupes/content/removedupes-dialog.js:611
    initDupeReviewDialog chrome://removedupes/content/removedupes-dialog.js:130
    _fireButtonEvent chrome://global/content/elements/dialog.js:487
    _doButtonCommand chrome://global/content/elements/dialog.js:466
    _handleButtonCommand chrome://global/content/elements/dialog.js:460
    _handleButtonCommand self-hosted:844
```

**Comment by eyalroz** [comment]
@sphh : Yes, this new error is a well-known issue (see e.g. [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1703503) - but it is harmless and unrelated. 

Closing this bug; if anyone still gets this failure with 0.5.2b3 - please reopen.

**Comment by sphh** [comment]
Thanks.

And sorry, I forgot to comment on the [obligatory restart of TB](https://addons.thunderbird.net/en-US/thunderbird/addon/removedupes/reviews/#review-1166173): When `removedupes` was updated automatically to 0.5.2**b2**, it could not find any duplicates. I had to follow the advice given in this [comment](https://addons.thunderbird.net/en-US/thunderbird/addon/removedupes/reviews/#review-1166173) to get it to display the duplicates. After I installed 0.5.2**b3**, this was not necessary. What I don't know, is if the restart was just on one-time necessity when upgrading from 0.5 to 0.5.**2x** or if it was a requirement for 0.5.2**b2** to work.
